leasing/report/lease/invoicing_disabled_report.py

```diff
@@ -33,11 +28,31 @@ class LeaseInvoicingDisabledReport(ReportBase):
     def get_data(self, input_data):
         today = timezone.now().date()
 
-        return Lease.objects.filter(
+        qs = Lease.objects.filter(
             Q(end_date__isnull=True) | Q(end_date__gte=today),
             start_date__isnull=False,
             state__in=[LeaseState.LEASE, LeaseState.SHORT_TERM_LEASE, LeaseState.LONG_TERM_LEASE],
             is_invoicing_enabled=False
         ).select_related(
             'identifier', 'identifier__type', 'identifier__district', 'identifier__municipality',
+        ).prefetch_related(
+            'rents',
         ).order_by('start_date', 'end_date')
+
+        aggregated_data = []
+        for lease in qs:
+            print("--")
+            free = True
+            for rent in lease.rents.all():
+                if rent.type is not RentType.FREE:
+                    free = False
+            if not free:
+                aggregated_data.append(
```

Comment:
You could rename the list to "leases" and just leases.append(lease). Then if later there is a need to add output_fields one doesn't need to add them here too.